package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "concat-text-webpack-plugin",
-  "version": "0.1.9",
+  "version": "0.1.10",
```

Comment:
We don't need to specify the version manually in the PR. This value will be updated once we prepare this for a release.